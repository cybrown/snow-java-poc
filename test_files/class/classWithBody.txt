class Foo {
    def a: int = 4;
    def aPlusOne = fun () {a + 1}
}
