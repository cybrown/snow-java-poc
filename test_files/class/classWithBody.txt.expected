<SnowClass>
    <Identifier>Foo</Identifier>
    <ExpressionList>
        <Definition>
            <Identifier>a</Identifier>
            <Identifier>int</Identifier>
            <LiteralInteger>4</LiteralInteger>
        </Definition>
        <Definition>
            <Identifier>aPlusOne</Identifier>
            <Function>
                <Binary operator="+">
                    <Identifier>a</Identifier>
                    <LiteralInteger>1</LiteralInteger>
                </Binary>
            </Function>
        </Definition>
    </ExpressionList>
</SnowClass>
